### ISSUE TITLE
Import images from customizer not working

### ISSUE THREAD

**Issue by rodica-andronache** [Codeinwp/ti-onboarding#42]
@abaicus like we've discussed, there is a problem importing images from customizer, as the path is not replaced correctly, now that we have optimole

**Comment by rodica-andronache** [comment]
@abaicus if you have time to have this fixed today, it would be great, as we have to release Hestia today ( there is another problem that's urgent ) Thanks 👍

**Comment by rodica-andronache** [comment]
Also, for this demo https://demo.themeisle.com/hestia-gym/, I realized the problem is that after the import, `hestia_about_hide` is false, but its a string, not boolean, like it should be.

I didn't manage to figure out why. Either that's coming from here as a string https://demo.themeisle.com/hestia-gym/wp-json/ti-demo-data/data or something happens on the import ( maybe related to the sync we have in the theme between the homepage and the about section in customizer ).

**Comment by rodica-andronache** [comment]
Another thing, don't think it's related to this, but don't know what's it, maybe something related to optimole. Just on the demos this happens https://prnt.sc/n4qcic, on local is fine

**Comment by abaicus** [comment]
@rodica-andronache The photos are there every time for me on the demos. 💃

**Comment by rodica-andronache** [comment]
ok, let me try again :)

**Comment by abaicus** [comment]
https://github.com/Codeinwp/ti-onboarding/pull/43
